src/host/main.js

```diff
@@ -68,6 +68,22 @@ export async function hostMain({
 
     const callbackRequests = [];
 
+    const postCallback = body => {
+      // ignore if not in callback mode
+      if (!callbackUrl) return;
```

Comment:
The name of this function is a bit misleading because invoking the function doesn't necessarily cause anything to be posted. I'd support `maybePostCallback` or keeping the more reasonable name and moving the conditional to each of the call sites.